blockbin/src/index.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
What's the difference between this and "app.css"? Seems like both of these should encompass everything!

Reply:
I think it might just be convention (things that are "global styles" would go in `body` and the things that you happen to want the parent div to have would be in `.app`. The `.app` css wasn't doing anything anyway though so I removed it :)